### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ArhanCodes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,12 +58,12 @@ The `examples` folder contains one framework example as of right now:
...
+> **Note**: Have an open-source bot of your own you'd like to feature? [Open an GitHub issue](https://github.com/CRBT-Team/Purplet/issues) and we may add it here!
 
 ## Contributing
 
 See the [contributing guide](CONTRIBUTING.md) for information on how we accept contributions, as well as an overview of the codebase.
 
 ## License
 
-Purplet is licensed under the [Apache License 2.0](https://github.com/CRBT-Team/Purplet/blob/main/LICENSE).
+Purplet is licensed under the Apache License 2.0. View the full file [here](https://github.com/CRBT-Team/Purplet/blob/main/LICENSE).
```
Comment (Clembs) [inline README.md:69]: It's generally considered bad practice to hide an hyperlink in a generic statement like "here" to "click here".

File: README.md [hunk 2]
```diff
@@ -28,8 +30,6 @@ export const helloWorld = $slashCommand({
 
 Our suite of packages is built from the ground up on Web Standards like `fetch` and `WebSocket`, allowing your code to run locally in Node.js, but with the ability to deploy your interaction handlers to Cloudflare Workers, greatly reducing costs while increasing scalability.
 
-> ⚠️ Purplet is extremely experimental software with a rapidly changing API. We do not recommend using it in production at this point. When v2.0 is released, the API will be stable, but it will still be considered experimental until we can ensure it's stability.
```
Comment (Clembs) [inline README.md:31]: Why not have this in the about section?

**Review by Clembs — COMMENTED** [review]
I don't understand the point of these changes

**Comment by paperdave** [comment]
im still recovering from being sick but whats the point of this pr. description is empty.

**Comment by Clembs** [comment]
> im still recovering from being sick but whats the point of this pr. description is empty.

they changed a few things in the readme file

**Comment by ArhanCodes** [comment]
just realized its useless

**Pull request closed without merging**